web/src/api/fbaAPI.ts

```diff
@@ -30,12 +43,20 @@ export interface ZoneAreaListResponse {
 
 export interface HfiThresholdFuelTypeArea {
   fuel_type_id: number
-  threshold: number
+  threshold_id: number
   area: number
 }
-export interface FireZoneThresholdFuelTypeResponse {
-  mof_fire_zone_id: number
-  fuel_types: HfiThresholdFuelTypeArea[]
+
+export interface HfiThreshold {
```

Comment:
Unused

Reply:
HfiThreshold is referenced in the `FireZoneThresholdFuelTypeArea` interface, which is being used in `hfiFuelTypesSlice`